docs/release-notes/breaking-changes.md

```diff
@@ -0,0 +1,36 @@
+---
+navigation_title: "Elastic APM Node.js Agent"
+---
+
+# Elastic APM Node.js Agent breaking changes [elastic-apm-nodejs-agent-breaking-changes]
+Before you upgrade, carefully review the Elastic APM Node.js Agent breaking changes and take the necessary steps to mitigate any issues.
+
+To learn how to upgrade, check out [Upgrading](/reference/upgrading.md).
+
+% ## Next version [elastic-apm-nodejs-agent-nextversion-breaking-changes]
+% **Release date:** Month day, year
+
+% ::::{dropdown} Title of breaking change
+% Description of the breaking change.
+% For more information, check [PR #](PR link).
+% **Impact**<br> Impact of the breaking change.
+% **Action**<br> Steps for mitigating deprecation impact.
+% ::::
+
+## 4.2.0 [elastic-apm-nodejs-agent-420-breaking-changes]
```

Comment:
I'll echo [Jonas's comments](https://github.com/elastic/apm-agent-java/pull/3981#discussion_r1983825763) that I'm not an immediate fan of separating out the breaking changes. It means that an upgrading user has potentially *two* places to check to find relevant changes in the new version. I think it would be easy to miss this breaking change in version 4.2.0.


Anyway, this isn't a blocking comment. Just opinion.

Reply:
cc @KOTungseth

Reply:
This is an interim state before we can implement filtering capabilities on our release docs pages. 

When we get to a standardized format, the goal is to provide users with the ability to filter their view for whatever products and versions they want.